### PR TITLE
feat: activate sentry for browser

### PR DESCRIPTION
**Pull request by asbiin**
This implements sentry for [browser](https://docs.sentry.io/platforms/javascript/).
As a lot of our code is in javascript/vue, it's important to follow issues in there.
Sentry provides an implementation for vue, but it's not vue3 ready yet.
Activation of [performance monitoring](https://docs.sentry.io/platforms/javascript/performance/) in the browser can be done by settings `SENTRY_TRACES_SAMPLE_RATE` > 0

- connect to sentry and vue in javascript
- pass sentry variables in app.blade.php
- fix sentry variable (which prevent sentry for working!)
- build source maps and send them to sentry (this does not affect assets builds)
- Add a new `SENTRY_DEFAULT_PII` environment variable to enable [Default Personally Identifiable Information](https://docs.sentry.io/platforms/php/data-management/sensitive-data/#personally-identifiable-information-pii) feature

You fool, don't forget these steps:

- [ ] Unit tests
- [ ] Tests with Cypress
- [ ] Documentation
- [ ] Dummy data

### CODE REVIEW

File: config/app.php [hunk 1]
```diff
@@ -123,13 +123,22 @@
...
-    | Enable sentry
-    |--------------------------------------------------------------------------
-    */
+    'sentry' => [
+        /*
+        |--------------------------------------------------------------------------
+        | Enable sentry
+        |--------------------------------------------------------------------------
+        */
+        'enabled' => (bool) env('SENTRY_ENABLED', false),
```
Comment (djaiss) [inline config/app.php:132]: Not sure we need to cast Boolean here.

**Review by djaiss — APPROVED** [review]
Awesome. 
￼￼with sentry we need to be extra careful not to send private and identifiable information so we won't have any privacy problems.

**Comment by asbiin** [comment]
> Awesome.
> ￼￼with sentry we need to be extra careful not to send private and identifiable information so we won't have any privacy problems.

I don't think that's an issue at all. Data on sentry are not available in public access, so I don't see any problem in pushing data there.

**Comment by djaiss** [comment]
> I don't think that's an issue at all.

It is, for GDPR reasons. The less personal data travel throughout all the systems, the better. Otherwise it'll be very complicated.

**Comment by asbiin** [comment]
> > I don't think that's an issue at all.
> 
> It is, for GDPR reasons. The less personal data travel throughout all the systems, the better. Otherwise it'll be very complicated.

The only data sent to sentry are defined in [sentry.js](https://github.com/officelifehq/officelife/blob/20210417-sentry/resources/js/sentry.js#L26) file, and are ids (+locale, but I don't think it's sensitive data).

**Pull request merged**